client/src/components/Home/MediaContainer.jsx

```diff
@@ -55,6 +57,12 @@ const MediaContainer = ({ token, weather }) => {
     if (data) setRecommendedTracks(data.tracks);
   };
 
+  // Function to get the user's recently played tracks
+  const getRecentlyPlayedTracks = async () => {
+    const data = await fetchWebApi("v1/me/player/recently-played?limit=25");
```

Comment:
Might be a good idea to move path prefix (v1/me/) to a variable, that way you can easy modify if something changes there.

Reply:
V1/me gets my information from sportify, if I need other people's information or other resources not owned by me it will need to be change according...